book/02-git-basics/sections/recording-changes.asc

```diff
@@ -606,6 +606,6 @@ $ git rm README.md
 $ git add README
 ----
 
-Git figures out that it's a rename implicitly, so it doesn't matter if you rename a file that way or with the `mv` command.
-The only real difference is that `git mv` is one command instead of three -- it's a convenience function.
-More importantly, you can use any tool you like to rename a file, and address the add/rm later, before you commit.
+Ang Git ay nakatukoy na ito ay isang pagpapahiwatig ng pagpalit ng pangalan, kaya hindi mahalaga na kung ikaw ay nakapalit ng pangalan sa file sa ganun na paraan o kasama ang `mv` na utos.
+Ang tanging tunay na kaibahan ay ang `git mv` ay isang utos sa halip na tatlo -- ito ay isang kaginhawahang function.
+Higit na mahalaga, maaari kang gumamit ng anumang kasangkapan na gusto mong ipalit na pangalan sa isang file, at maipasa sa add/rm mamaya, bago ka mag-commit.
```

Comment:
or maaari kang gumamit ng anumang kasangkapan na gusto mo upang magpalit na pangalan sa isang file

Reply:
done change kaginhawahang to kaginhawaang
and also the sentence